src/util/audioApi.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import {
+  resolveJsonOrRejectWithError,
+  apiResourceUrl,
+  fetch,
+} from 'util/apiHelpers';
+
+const baseUrl = apiResourceUrl('/audio-api/v1/series');
```

Comment:
Eg ville aller helst latt all kommunikasjon fra ndla-frontend gå via graphql. Altså unngå direktekommunikasjon til forskjellige apis. Kva tenker du?

Reply:
Ser ikke helt hva det gir oss annet enn en vanskelig å lese POST-request når vi bare skal kontakte et api uansett.
Vi kan vel heller ikke bruke `useQuery` hook'en siden det ikke skjer i en react-komponent (og server-side for that matter).
Så da blir det vel noe sånt.

```
POST /graphql-api/graphql
{"query":"{ podcastSeries(id: \"${seriesId}\") { id title description created coverPhoto audioFile { url fileSize mimeType } episodes { copyright podcastMeta { coverPhoto introduction } } }}"}
```

vs 

```
GET /audio-api/v2/series/${seriesId}
```

Hvorfor vil vi unngå direktekommunikasjon egentlig?

Reply:
Eg berre tenker det er reinere/ryddigere. Men ser poenget ditt. Det er enklere å lese slik du har gjort det.